### PR TITLE
feat(y-indexeddb): cleanup

### PR DESCRIPTION
**Pull request by sxzz**

### CODE REVIEW

**Review by Himself65 — COMMENTED** [review]
Could you please add a test case for the update?

File: packages/y-indexeddb/src/index.ts [hunk 1]
```diff
@@ -274,8 +274,8 @@ export const createIndexedDBProvider = (
       doc.off('update', handleUpdate);
       doc.off('destroy', handleDestroy);
     },
-    cleanup() {
-      // todo
+    async cleanup() {
+      (await dbPromise).delete('workspace', id);
```
Comment (himself65) [inline packages/y-indexeddb/src/index.ts:287]: There might have some edge case.

For example, it should throw an error when the provider is still connecting.

**Review by Himself65 — CHANGES_REQUESTED** [review]

**Review by Himself65 — APPROVED** [review]

File: packages/y-indexeddb/src/index.ts [hunk 2]
```diff
@@ -275,6 +281,9 @@ export const createIndexedDBProvider = (
       doc.off('destroy', handleDestroy);
     },
     async cleanup() {
+      if (connect) {
```
Comment (sxzz) [inline packages/y-indexeddb/src/index.ts:284]: I'm a bit confused. Is `connect` for connecting or connected?

Comment (himself65) [reply packages/y-indexeddb/src/index.ts]: connected

**Pull request merged**